src/app/pages/Splashpage/Splashpage.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './Splashpage.module.css';
+
+type SplashpageProps = {
+  imgSrc: string;
+};
+
+function Splashpage({ imgSrc }: SplashpageProps): JSX.Element {
+  return (
+    <div className={styles.splash}>
+      <main>
```

Comment:
You can remove the outer div if you like
```suggestion
    <main className={styles.splash}>
```

Reply:
You're right. I removed it. Ty!

Reply:
@fgottesmann :),  you should remove it in this pull request before merging next time